packages/benchmarks/src/reporting/EvalCases.ts

```diff
@@ -0,0 +1,126 @@
+import { ObjectId } from "mongodb-rag-core/mongodb";
+import { OpenAI } from "mongodb-rag-core/openai";
+
+type ChatCompletionMessageParam = OpenAI.Chat.ChatCompletionMessageParam;
+
+/**
+ This represents a single prompt that we pass to models for evaluation.
+*/
+export type BaseCase<
+  CaseMetadata extends Record<string, unknown> = Record<string, unknown>,
+  CaseResult extends Result = Result
+> = {
+  /*
+  A unique identifier for the prompt.
+  */
+  id: ObjectId;
```

Comment:
```suggestion
  _id: ObjectId;
```

Let's make this an oid - will store as a document in a collection